packages/react-scripts/template/public/manifest.json

```diff
@@ -3,9 +3,19 @@
   "name": "Create React App Sample",
   "icons": [
     {
-      "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
```

Comment:
Can we not just add additional sizes here? That seems to be the solution in the issue you linked to. We also probably don't want to remove these smaller icon sizes.